src/surfaces/modal/modalHeader.jsx

```diff
@@ -0,0 +1,146 @@
+        root: PropTypes.string,
+        title: PropTypes.string,
+    }),
+    closeButton: PropTypes.oneOfType([
+        PropTypes.bool,
+        PropTypes.shape({}),
+        PropTypes.string,
+    ]),
+    inverse: PropTypes.bool,
+    onClose: PropTypes.func,
+    title: PropTypes.string,
+    titleTruncate: PropTypes.bool,
+};
+
+const defaultProps = {
+    children: null,
+    classes: null,
+    closeButton: null,
+    inverse: null,
+    onClose: null,
+    title: null,
+    titleTruncate: null,
+};
+
+const useStyles = makeStyles((theme) => ({
+    closeButtonContainer: {
+        alignItems: 'center',
+        display: 'flex',
+        flex: '0 1 1px',
+        height: 29,
+        justifyContent: 'center',
+        marginBottom: 0,
+        width: 16,
+        [theme.breakpoints.up('md')]: {
+            height: 33,
+            width: 33,
+        },
+    },
+    root: {
+        alignItems: 'center',
+        backgroundColor: theme.palette.background.contrastPrimary,
+        display: 'flex',
+        flexWrap: 'wrap',
+        justifyContent: 'flex-start',
+        left: 0,
+        padding: `${theme.spacing(3)}px 11px`,
+        position: 'fixed',
+        top: 0,
+        width: '100%',
+        '& .ui.divider': {
+            flex: '0 1 100%',
+            margin: '11px 0 0',
+        },
+        [theme.breakpoints.up('md')]: {
+            backgroundColor: theme.palette.background.primary,
+            padding: `${theme.spacing(3)}px ${theme.spacing(3)}px 22px`,
+        },
+    },
+    title: {
+        alignItems: 'flex-start',
+        display: 'flex',
+        flex: '1 1 1px',
+        justifyContent: 'flex-start',
+        margin: 0,
+        paddingRight: theme.spacing(3),
+        width: '100%',
+    },
+}));
+
+function ModalHeader(props) {
+    const {
+        children,
+        closeButton,
+        inverse,
+        onClose,
+        title,
+        titleTruncate,
+    } = props;
+
+    const classes = useStyles();
+
+    const onCloseClick = () => {
+        onClose();
+    };
+
+    return (
+        <header className="modal-header">
+            <Header
+                as="h3"
+                className={ClassNames(
+                    classes.title,
+                    {
+                        'modal-title-truncate': titleTruncate,
+                    },
+                )}
+                title={title}
+                weight="bold"
+            >
+                {title}
+            </Header>
+
+            <div className={classes.closeButtonContainer}>
+                {(isUndefined(closeButton) || isString(closeButton)) && (
+                    <Button
+                        className="modal-close-button"
+                        color={inverse ? 'transparent' : 'alternate'}
+                        onClick={onCloseClick}
+                        icon
+                    >
+                        <Icon inverse type={isString(closeButton) ? closeButton : 'times'} />
```

Comment:
`title={false}` (or `title="Close"`)

Reply:
I'm not seeing a `title` prop here either. Not a blocker, but nice to have (IMHO).

Reply:
Added title to Button.

Reply:
There we go.  Resolving.